lib/templateMetaFactory.js

```diff
@@ -23,18 +23,49 @@ class TemplateMetaFactory extends BaseFactory {
         return new TemplateMeta(config);
     }
 
+    /**
+     * Create pipeline template meta
+     * @method list
+     * @param  {Object}   config                  Config object for template meta
+     * @param  {Object}   config.pipelineId       Identifier of the pipeline that is publishing the template
+     * @return {Promise}                          Resolve pipeline template meta
+     */
     create(config) {
         config.templateType = this._getTemplateType();
 
         return super.create(config);
     }
 
+    /**
+     * Get pipeline template meta
+     * @method list
+     * @param  {Object}   config                  Config object for template meta
+     * @param  {Object}   config.params           Parameters to filter on
+     * @return {Promise}                          Resolve pipeline template meta
+     */
     get(config) {
         config.templateType = this.getTemplateType();
 
         return super.get(config);
     }
 
+    /**
+     * List all pipeline template meta
+     * @method list
+     * @param  {Object}   config                  Config object for template meta
+     * @param  {Object}   config.params           Parameters to filter on
+     * @return {Promise}                          Resolve pipeline template meta list
+     */
+    list(config) {
+        config.params.templateType = this.getTemplateType();
+
+        return super.list(config);
```

Comment:
We should do something similar what is being done in `[TemplateTagFactory](https://github.com/screwdriver-cd/models/blob/master/lib/templateTagFactory.js#L149)`
```suggestion
        if (!config.params) {
            config.params = {};
        }
        config.params.templateType = this.getTemplateType();

        return super.list(config);
```